test/format/shortfnsyntax.d

```diff
@@ -0,0 +1,11 @@
+int bingBong(int x) => x + 1;
+
+int _______________________(int x)
+	=> x + 1;
```

Comment:
That's good. We also need a test case where the expression in there splits.

Reply:
Added something that breaks.

Reply:
Looks ok.

Reply:
Yes, this looks good. One thing I have though of since is that we maybe break after the type, in which case I think we do not indent. The way the span are layed out might break this, so it's a good idea to check.